tests/unit/customizations/test_cloudtrail.py

```diff
@@ -144,14 +158,41 @@ def test_s3_create_set_policy_fail(self):
 
         s3.PutBucketPolicy = orig
 
-    def test_get_policy_fail(self):
-        orig = self.subscribe.s3.GetObject
+    def test_s3_get_policy_fail(self):
         self.subscribe.s3.GetObject = Mock(side_effect=Exception('Error!'))
 
         with self.assertRaises(Exception):
             self.subscribe.setup_new_bucket('test', 'logs')
 
-        self.subscribe.s3.GetObject = orig
+    def test_s3_get_policy_logs_messages(self):
+        cloudtrail.LOG = Mock()
```

Comment:
This is never restored.  A unit test shouldn't be modifying global state after its execution.